be/src/exec/hdfs_scanner_text.cpp

```diff
@@ -385,6 +385,14 @@ Status HdfsTextScanner::_create_or_reinit_reader() {
 }
 
 Status HdfsTextScanner::_build_hive_column_name_2_index() {
```

Comment:
Could you let me know why you changed this function?
It seems this doesn't relate to the snappy compression.

Reply:
And better to give some comments for this newly added logic

Reply:
It's only for test. When we use file table for testing, there is no hive names. 

And when there is no hive names, we use slot orders. (And that's also the logic of orc scanner).

```
CREATE EXTERNAL TABLE test_csv_snappy_format
(
    a string,
    b string
) 
ENGINE=file
PROPERTIES 
(
    "path" = "${snappy_prefix[1]}",
    "format" = "text",
    "column_separator" = ","
);
```

Reply:
I'll add comment later.